src/main/scala/Main.scala

```diff
@@ -1,5 +1,69 @@
-object Main extends App{
+import java.io._
+
+import com.github.tototoshi.csv.{CSVReader, CSVWriter}
+
+import scala.collection.parallel.ParSeq
+import scala.io.BufferedSource
+
+/**
+  * Program to clean up the data
+  */
+object Main{
+
+  val separator = System.lineSeparator()
+
+  def main(args: Array[String]): Unit = {
+
+    println("Main app for cleaning DB")
+
+    //IO Paths
+    val filePath = "/home/hedi/Documents/EPFL/sem_6/intro_db/DB_project/dataset/barcelona_listings.csv"
```

Comment:
maybe change it to ../dataset/... and ../cleanedData/..., so we don't have to change the path every time we pull from the master branch